Utility.h

```diff
@@ -1,17 +1,19 @@
 #pragma once
 
+#include <string>
 #include <vector>
 #include <memory>
-#include "Item.h"
-
 
+struct Item;
```

Comment:
Remove this forward declaration
- [ ] REVISE

Reply:
done